src/mm/dbcsr_mm_accdrv.F

```diff
@@ -566,6 +566,7 @@ SUBROUTINE dbcsr_mm_accdrv_process(this, left, right, params, stack_size, &
       CALL acc_event_record(stackbuf%ready, stream=upload_stream)
 
       CALL acc_stream_wait_event(stackbuf%stream, stackbuf%ready)
+      CALL acc_stream_wait_event(c_area%memory_type%acc_stream, stackbuf%ready)
```

Comment:
I don't think this is needed because `libsmm_acc_process_blas` uses `param_stack_host`. Hence, it does not have to wait for them to be uploaded to the device.

Reply:
Let's keep the discussion here...
Your point is correct, there is not need to wait for the upload of the stacks.
Can it be that cublas has to start after the transpose kernel? How to sync that?